javascripts/app.js

```diff
@@ -30,6 +29,62 @@ console.log("spell: ", spell.toString());
    */
   $("#player-setup").show();
 
+  // Display Name
+  $("#nameChoice").on("click", function() {
+    warrior.name = $("#player-name").val();
+    $("#battleName").append(warrior.name);
+  });
+
+  $(".classChoice").on("click", function() {
+    var humanClass = this.id
+
+    switch(humanClass) {
```

Comment:
This switch statement is completely unnecessary. It can be replaced with a 4 line `if` statement.

```js
if (this.id === "surprise") {
  // do something
} else {
  // do something else
};
```